plaso/containers/sessions.py

```diff
@@ -185,8 +237,8 @@ def __init__(self, identifier=None):
     self.timestamp = None
 
 
-class SessionStart(interface.AttributeContainer):
-  """Session start attribute container.
+class SessionConfiguration(interface.AttributeContainer):
+  """Session configuration attribute container.
```

Comment:
Please add a description of what a session configuration is, and what sort of information should be stored in one.

Reply:
done